telematic_system/telematic_historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/service_manager.py

```diff
@@ -116,26 +119,65 @@ async def get_file_path_from_nats(self, msg):
         data = msg.data.decode()
         msg_json_object = json.loads(data)
 
-        rosbag_name = msg_json_object["filename"]
+        rosbag_path = msg_json_object["filepath"]
 
-        rosbag_path = Path(self.config.upload_destination_path) / rosbag_name
         # Add rosbag name to queue
         self.rosbag_queue.append(rosbag_path)
 
-    async def process_rosbag(self):
+    async def process_rosbag_queue(self):
         # This task is responsible for processing the rosbag in the queue - As long as the queue is not empty - keep processing
         while True:
             if not self.rosbag_parser.is_processing and self.rosbag_queue:
-                self.rosbag_parser.process_rosbag(self.update_first_rosbag_status())
+                # Get the mysql entry name: org-name/
+                rosbag_complete_path = Path(self.rosbag_queue[0])
+                rosbag_mysql_entry = str(rosbag_complete_path.relative_to(Path(self.config.upload_destination_path)))
```

Comment:
Which lib is this relative_to from? Does this return a substr of the absolute path?

Reply:
can we rename this "rosbag_mysql_entry" with "org_filename" or something that indicate it is a filename string?

Reply:
That relative_to function is extracting "my-org./bag_file_name" from "/opt/telematics/upload/my-org./bag_filename"